src/init.cpp

```diff
@@ -866,6 +866,9 @@ void ThreadImport(std::vector <boost::filesystem::path> vImportFiles) {
     if (!GetBoolArg("-disablewallet", false)) {
         zwalletMain->SyncWithChain();
     }
+    if (GetBoolArg("-zapwallettxes", false)) {
```

Comment:
Is this for debugging?

Reply:
No, if `zapwallettxes` is used it zaps the mempool, so we need to perform an update of the meta objects. If you look at `InitLoadWallet` is does something similar - it performs an update of the status mint objects if `zapwallettxes` is used